privacyidea/lib/user.py

```diff
@@ -306,6 +306,8 @@ def info(self):
             return {}
         (uid, _rtype, _resolver) = self.get_user_identifiers()
         y = get_resolver_object(self.resolver)
+        if uid = None:
+            return None
```

Comment:
I think we should return an empty dict like in line 306, since this method is supposed to return a dict.

Reply:
Done.